data/json/music.json

```diff
@@ -0,0 +1,712 @@
+[
+  {
+    "type": "snippet",
+    "category": "musicgenre_a",
+    "text": [ "popular " ]
+  },
+  {
+    "type": "snippet",
+    "category": "musicgenre_a",
+    "text": [ "top-forty " ]
+  },
+  {
+    "type": "snippet",
+    "category": "musicgenre_a",
+    "text": [ "coorporate " ]
+  },
+  {
+    "type": "snippet",
+    "category": "musicgenre_a",
+    "text": [ "commercial " ]
```

Comment:
This is an array. You don't need to put only one item into it, it can have as many items as you want. You don't need so many separate snippet defintions for the same category.